lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -343,6 +352,13 @@ class DomHTMLDivElement extends DomHTMLElement {}
 DomHTMLDivElement createDomHTMLDivElement() =>
     domDocument.createElement('div') as DomHTMLDivElement;
 
+@JS()
+@staticInterop
+class DomHTMLButtonElement extends DomHTMLElement {}
+
+DomHTMLButtonElement createDomHTMLButtonElement() =>
+    domDocument.createElement('button') as DomHTMLButtonElement;
```

Comment:
Would the static introp allow this to be a static method on `DomHTMLButtonElement`?

Reply:
Alas, originally I had done just that, but static methods on staticInterop classes were broken in DDC at the time. This might be fixed now, though I have to check, but we might have issues when we move to views.

If you still want to take a chance on static methods, then is it okay if we wait until the end the last migration CL to give it a shot? I can leave it as a final TODO.